object-construction-checker/tests/basic/ACExceptionalExitPointTest.java

```diff
@@ -0,0 +1,38 @@
+import org.checkerframework.checker.objectconstruction.qual.*;
+import org.checkerframework.common.returnsreceiver.qual.*;
+
+class ACExceptionalExitPointTest {
```

Comment:
Can we move these `AC*` tests to a folder `object-construction-checker/tests/alwayscall` and then run them in a separate test class?